packages/component/src/Utils/RelativeTime.js

```diff
@@ -1,68 +1,50 @@
 /* eslint react/no-unsafe: off */
```

Comment:
Could you do a broad search on keyword `react/no-unsafe` and remove them? And file bugs for those we cannot remove as we didn't migrate yet.

Reply:
done. #2345 and #2313